bigquery/storage/managedwriter/options.go

```diff
@@ -48,19 +49,23 @@ type writerClientOption interface {
 }
 
 // enableMultiplex enables multiplex behavior in the client.
+// maxSize indicates the maximum number of shared multiplex connections
+// in a given location/region
 //
 // TODO: export this as part of the multiplex feature launch.
-func enableMultiplex(enable bool) option.ClientOption {
-	return &enableMultiplexSetting{useMultiplex: enable}
+func enableMultiplex(enable bool, maxSize int) option.ClientOption {
```

Comment:
there is going to be a recommended maxSize here ? Users might ask what number to use here when this is exported.

Reply:
When this gets exported I expect it to be two options, and both will start as being marked experimental.  We'll also want to provide guidance about controlling the grpc connection options as well, and can likely adopt similar guidance around connection an GOMAXPROCS or similar.